.github/workflows/tests+pypi.yml

```diff
@@ -244,6 +244,7 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           files: |
             /github/home/figures/*.svg
+            /github/home/figures/*.gif
```

Comment:
this is a valid change, but the error on `main` branch seems to happen in the previous step